### PR TITLE
Add memorial and photograph search displays

### PR DESCRIPTION
**Pull request by kehh**
* Also add restrictions and group access permissions for library and museum search displays
* whitespace reformatting

### CODE REVIEW

File: profile/rwahs.xml [hunk 1]
```diff
@@ -7814,6 +7822,9 @@
           <name>Library Search display</name>
         </label>
       </labels>
+      <typeRestrictions>
+        <restriction type="library"/>
```
Comment (leftclickben) [inline profile/rwahs.xml]: Makes sense

File: profile/rwahs.xml [hunk 2]
```diff
@@ -7924,6 +7947,113 @@
...
+        <permission group="museum" access="read"/>
+        <permission group="photograph" access="read"/>
+        <permission group="memorial" access="read"/>
+      </groupAccess>
+      <bundlePlacements>
+        <placement code="ca_objects_preferred_labels">
+          <bundle>ca_objects.preferred_labels</bundle>
+          <settings>
+            <setting name="label" locale="en_AU">Title</setting>
+            <setting name="format"><![CDATA[<div style="min-width:30em;">^preferred_labels</div>]]></setting>
```
Comment (leftclickben) [inline profile/rwahs.xml]: I can see this has been copied from the existing displays, but should these settings, in particular `format`, be here?  Wouldn't it give what we want if only the `label` and `maximum_width` settings were given here?

File: profile/rwahs.xml [hunk 3]
```diff
@@ -7924,6 +7947,113 @@
...
+        <permission group="museum" access="read"/>
+        <permission group="photograph" access="read"/>
+        <permission group="memorial" access="read"/>
+      </groupAccess>
+      <bundlePlacements>
+        <placement code="ca_objects_preferred_labels">
+          <bundle>ca_objects.preferred_labels</bundle>
+          <settings>
+            <setting name="label" locale="en_AU">Name of memorial</setting>
+            <setting name="format"><![CDATA[<div style="min-width:30em;">^preferred_labels</div>]]></setting>
```
Comment (leftclickben) [inline profile/rwahs.xml]: Same as above re `settings`.

**Comment by leftclickben** [comment]
Just the question re `settings`.

File: profile/rwahs.xml (hunk 2 above)
Comment (kehh) [inline profile/rwahs.xml]: This is where the title was too long making the table look funny (don't have time to find the issue number but it's in youtrack), and this is the fix that you put in for the display ;) Needs both

Comment (leftclickben) [inline profile/rwahs.xml]: Hmmm... (points fingers in air) pew pew

Comment (leftclickben) [inline profile/rwahs.xml]: Not one of my best... Probably why I'd forgotten about it.

**Pull request merged**